packages/astro/src/core/build/static-build.ts

```diff
@@ -201,7 +201,7 @@ async function clientBuild(
 	// TODO: use vite.mergeConfig() here?
 	info(opts.logging, null, `\n${bgGreen(black(' building client '))}`);
 
-	const viteBuildConfig = {
+	const viteBuildConfig = vite.mergeConfig({
 		logLevel: 'info',
 		mode: 'production',
 		css: viteConfig.css,
```

Comment:
probably not important, but can `css`, `server`, and `base` be removed here since they'll get merged in from `viteConfig`?